.pre-commit-hooks.yaml

```diff
@@ -35,15 +35,7 @@
 - id: flake8-lint
   name: flake8-lint
   description: '`flake8` is a command-line utility for enforcing style consistency across Python projects.'
-  entry: flake8  
+  entry: 'flake8 --ignore=E501'
```

Comment:
Why do we want to completely ignore E501? Cannot we define a specific number of characters within a line?

Reply:
what number is in your mind ? Right now we can go with the max no.of chars we observed to fix.

Reply:
> what number is in your mind ? Right now we can go with the max no.of chars we observed to fix, but that keep changes.

I would prefer 120 characters max.